packages/flutter/test/material/dropdown_test.dart

```diff
@@ -1326,61 +1326,56 @@ void main() {
         TestSemantics.rootChild(
           children: <TestSemantics>[
             TestSemantics(
+              flags: <SemanticsFlag>[
```

Comment:
GitHub did an amazingly poor job of diffing t
his.  It's basically removing an empty node at the end, and an empty layer in the tree.

![Screen Shot 2022-08-17 at 3 56 17 PM](https://user-images.githubusercontent.com/8867023/185257754-bec46f4e-f33b-4728-a440-a31172fa7f1d.png)

Reply:
I recommend reviewing this with githubs "Hide whitespace" option. Makes it a lot better!